src/constants.js

```diff
@@ -24,23 +28,30 @@ const displayMedium = `500 ${displayFontSize}/${displayLineHeight} ${displayFont
 
 const brightGreen = "#25b894";
 const darkGreen = "#005450";
+const darkerGreen = "#00413E";
+const darkGray = "#5A6575";
+const lightGray = "#ECEDEF";
 const white = "#fff";
 
 export const THEME = {
   colors: {
     // descriptive
     blue: "#3e8df7",
     darkGreen,
+    darkerGreen,
     // functional
     background: "#FCFCFC",
     body: "#707F96",
     bodyLight: white,
-    controlBackground: "#ECEDEF",
+    controlBackground: lightGray,
     controlLabel: "#3D4045",
-    controlValue: "#5A6575",
+    controlValue: darkGray,
+    footer: hexAlpha("#E5ECEC", 0.6),
```

Comment:
confession time: I have been converting almost all of the transparencies in the Figma mocks to their RGB equivalents ... unless something has a functional reason to be transparent. I guess I am not super committed to that as an ongoing strategy but it made sense to me at the time? anyway I mention this because I think we should try to at least be consistent one way or another. might be something we should check with Juan about actually

Reply:
Ok, I posted the question to Juan but in the meantime will just convert to the RGB equivalent and remove the `hex-alpha` dependency